reflections/2020-05-30 - kerry edits.md

```diff
@@ -0,0 +1,43 @@
+# Reflections
+
+## Participation
+
+- 3 groups 6-8 people in size
+
+## What Happened
+
+- shared screen from the beginning to show google colab notebook
+- opened the floor for questions and got a few questions from
+  participants
+- multiple participants had questions about what a csv is, and what a delimiter
+  refers to
+  - one group pulled up the csv in a text editor to show the contents
+- how to choose which way to read in files
+  (is all my data numeric, or strings, etc.)
+  influences whether numpy is the appropriate choice
+- slicing that removes a dimension versus slices that keep all dimensions was
+  a confusing topic
+- questions about proper use cases for using slices
+- introduce considerations of the amount of data you’re working on
+  (example was voxel arrays)
+- trade-offs among storing intermediate results in memory, vs recomputing
+  time-consuming intermediate results when they would have fit in memory,
+- illustrate the coding style tradeoffs between nesting/composition
+  and sequentially saving results
+- increased interest in the shell commands at the top of the notebook
+- lots of emphasis placed on axis=0 being rows, and axis=1 being columns
+- talked about position of arguments in a function, and how
+  changing the order created errors
+- one group did a round robin to ask questions and it appeared
+  to help with engagement (cold calling does not work for all learners however)
+
+## Take-Away(s)
+
+- keep sharing screen from beginning
+- demonstrate working with data in a familiar interface (excel)
+  before moving to python
+- try a mix of asking the crowd and doing round robin question asking
+  to hear from everyone? (or have another strategy to hear from everyone,
+  like a quota?)
+-Encourage participants to practice coding outside of CoLab.
+--It might be beneficial to remind/encourage participants to type code into cell instead of just running code or copy and pasting code.
```

Comment:
```suggestion
- It might be beneficial to remind/encourage participants to type code into cell instead of just running code or copy and pasting code.
```
dash and space for this list item.